### PR TITLE
For laravel 8

### PR DESCRIPTION
**Pull request by masashi1014**

### CODE REVIEW

**Comment by masashi1014** [comment]
@ikari7789 
Sorry I created new PR for some reasons..
Please review it again..

File: composer.json [hunk 1]
```diff
@@ -13,14 +13,15 @@
...
+        "guzzlehttp/guzzle": "^6.3|^7.0",
+        "illuminate/support": "~5.0|~6.0|~7.0|~8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
+        "phpunit/phpunit": "^6.0|^7.0|^8.0",
         "squizlabs/php_codesniffer": "^3.0",
-        "orchestra/testbench": "~3.0",
-        "mockery/mockery": "^0.9.7"
+        "orchestra/testbench": "^6.0",
```
Comment (ikari7789) [inline composer.json]:
```suggestion
        "orchestra/testbench": "^3.0|^4.0|^5.0|^6.0",
```

File: composer.json [hunk 2]
```diff
@@ -13,14 +13,15 @@
...
+        "mockery/mockery": "^1.3.0",
+        "composer/composer": "^2.0"
```
Comment (ikari7789) [inline composer.json]: Why are we requiring composer?

**Review by ikari7789 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (masashi1014) [reply composer.json]: Sorry we don't need it. I removed.

**Review by ikari7789 — APPROVED** [review]

**Comment by ikari7789** [comment]
I think the build pipeline should be expanded slightly, but I would fix it in a separate PR as it could break things unrelated to the aim of this PR. Looks good overall. Thank you.

File: .github/workflows/php.yml [hunk 3]
```diff
@@ -1,37 +1,136 @@
...
+        with:
+          path: "${{ steps.determine-composer-cache-directory.outputs.directory }}"
+          key: "php-${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+
+      - name: "Run squizlabs/php_codesniffer"
+        run: "vendor/bin/phpcs --standard=PSR2 src"
```
Comment (kevincobain2000) [inline .github/workflows/php.yml]: Change to 

```
vendor/bin/phpcs --standard=PSR2 src -n
```

```
 -n    Do not print warnings (shortcut for --warning-severity=0)
```

**Review by kevincobain2000 — APPROVED** [review]

**Pull request merged**